### PR TITLE
Tidy `README.md`, fix nitpicks and add badges

### LINKED ISSUE #141: Update README.md
# This is a Feature Proposal

## Description

There's a few minor nitpicks in the `README.md` file. It would also be nice to have badges for NPM etc. for getting project status quickly.

### PR DESCRIPTION
**Pull request by hassankhan**
## What did you implement:

Closes #141

## How did you implement it:

Updated `README.md`.

## How can we verify it:

Viewing `README.md`.

## Todos:

- [ ] ~~Write tests~~
- [X] Write documentation
- ~~Fix linting errors (not yet implemented)~~
- ~~Make sure code coverage hasn't dropped (not yet implemented)~~
- [ ] ~~Provide verification config / commands / resources~~
- [X] Enable "Allow edits from maintainers" for this PR
- [X] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
 # Serverless Webpack
 
-[![serverless](http://public.serverless.com/badges/v3.svg)](http://www.serverless.com)
-[![CircleCI](https://circleci.com/gh/elastic-coders/serverless-webpack.svg?style=shield)](https://circleci.com/gh/elastic-coders/serverless-webpack)
+[![Serverless][ico-serverless]][link-serverless]
+[![CircleCI][ico-circleci]][link-circleci]
+[![NPM][ico-npm]][link-npm]
```
Comment (HyperBrain) [inline README.md]: [npm version](https://badge.fury.io/js/serverless-webpack)

Comment (HyperBrain) [inline README.md]: Oops ... see my plugin for the source ;-)

Comment (HyperBrain) [inline README.md]: `[npm](https://www.npmjs.com/package/serverless-webpack)`

Comment (hassankhan) [inline README.md]: The icons are already SVG unless I'm mistaken... if you check the bottom of `README.md`:

```
[ico-serverless]: http://public.serverless.com/badges/v3.svg
[ico-circleci]: https://img.shields.io/circleci/project/github/elastic-coders/serverless-webpack.svg
[ico-npm]: https://img.shields.io/npm/v/serverless-webpack.svg
[ico-contributors]: https://img.shields.io/github/contributors/elastic-coders/serverless-webpack.svg
```

Comment (HyperBrain) [inline README.md]: ;) Sorry ... did not read until the end.

**Pull request merged**